pom.xml

```diff
@@ -562,15 +560,41 @@
                     </executions>
                 </plugin>
                 <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-remote-resources-plugin</artifactId>
+                    <version>1.5</version>
+                    <dependencies>
+                        <dependency>
+                            <groupId>org.hibernate.validator</groupId>
+                            <artifactId>hibernate-validator-build-config</artifactId>
+                            <version>${project.version}</version>
+                        </dependency>
+                    </dependencies>
+                    <configuration>
+                        <resourceBundles>
+                            <resourceBundle>org.hibernate.validator:hibernate-validator-build-config:${project.version}</resourceBundle>
+                        </resourceBundles>
+                        <!--<outputDirectory></outputDirectory>-->
```

Comment:
Mmmh maybe we could use a directory which makes more sense, such as build-config-resources or something similar?